docs/components/IssueCredentialOptions.yml

```diff
@@ -42,6 +42,10 @@ components:
             type:
               type: string
               description: The type of credential status to issue the credential with
+        revocable: 
+          type: string
+          enum: ["revocable", "non-revocable"]
```

Comment:
Why not just a `true`/`false` value here?

Reply:
How is this intended to work with `credentialStatus` -- doesn't that indicate whether or not the VC is revocable?

Reply:
right now, you are required to submit a credential with a member of the following form:

```
    "credentialStatus": {
      "id": "https://issuer.oidp.uscis.gov/credentials/status/3#94567",
      "type": "RevocationList2020Status",
      "revocationListIndex": "0",
      "revocationListCredential": "https://w3c-ccg.github.io/vc-http-api/fixtures/revocationList.json"
    },
```

perhaps it would be better to define this shape, and note that when it is present the credential MUST have an `id`.

Using the API today, you can issue revocable credentials, you just need to know how to hand crafter perfect JSON-LD to do it...

Reply:
I guess the question this raises, is... is the issuer really doing issuance, if they are not checking that the credentialStatus is wellformed? why are folks accepting / issuing vc's with `credentialStatus` that point to a CCG github repo?

I would expect an issuer to maintain their own list, or at the very least validate this entry from a schema perspective.